packages/story-editor/src/components/panels/design/layer/layer.js

```diff
@@ -31,41 +36,80 @@ import { LayerText } from '../../../../elements/shared/layerText';
 import useLayerSelection from './useLayerSelection';
 import { LAYER_HEIGHT } from './constants';
 
+const ActionsContainer = styled.div`
+  position: absolute;
+  display: none;
+  align-items: center;
+  height: 100%;
+  top: 0;
+  right: 0;
+  padding-right: 6px;
+
+  --background-color: ${({ theme }) =>
+    theme.colors.interactiveBg.secondaryNormal};
+  background-color: var(--background-color);
+  box-shadow: 0px 0px 15px 20px var(--background-color);
```

Comment:
```suggestion
  ${({ theme }) => `
  background-color: ${theme.colors.interactiveBg.secondaryNormal};
  box-shadow: 0px 0px 15px 20px ${theme.colors.interactiveBg.secondaryNormal};
  `
```

We don't usually use CSS custom properties in our styled components, especially not when it's just to save some characters.

Reply:
OK, I see you're using the custom property further below for hover&active state. But perhaps you could just use this way of writing there too.

Reply:
IMO, there's less room for error with the css var approach because if color assignment ever updates, with the ccs var we only have to update 1 css property vs targeting every property and child property that consumes it.

Consider the actions down below that consume this `--background-color` to create the relevant focus styles.

What's the aversion to the css var here? I see some benefits, but I'm struggling to see the negatives of using it

Reply:
No aversion. I just didn't see that you used it in so many places and thought it was only here, in which case it was quite unnecessary.

But for this use case I suppose it's useful 👍